macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -6,134 +6,340 @@
 import AppKit
 import FluentUI
 
+@available(OSX 10.14, *)
```

Comment:
If we bump the min os of the test app, can we not use these tags?

Reply:
Fixed. I must have picked up a change to the minimum OS version recently and was able to remove this stuff.